test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetAddPackageTests.cs

```diff
@@ -613,5 +613,268 @@ public async Task AddPkg_WhenPackageSourceMappingConfiguredCanotInstallsPackages
+                    pathContext.PackageSource,
+                    PackageSaveMode.Defaultv3,
+                    packageX200);
+
+            var propsFile = @$"<Project>
+                                <PropertyGroup>
+                                <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+                                </PropertyGroup>
+                                <ItemGroup>
+                                <PackageVersion Include=""X"" Version=""1.0.0"" />
+                                </ItemGroup>
+                            </Project>
+                            ";
+
+            solution.Projects.Add(projectA);
+            solution.Create(pathContext.SolutionRoot);
+
+            File.WriteAllText(Path.Combine(pathContext.SolutionRoot, "Directory.Packages.props"), propsFile);
+            var projectADirectory = Path.Combine(pathContext.SolutionRoot, projectA.ProjectName);
+
+            //Act
+            var result = _fixture.RunDotnet(projectADirectory, $"add {projectA.ProjectPath} package {packageX} -v {version}", ignoreExitCode: true);
+
+            // Assert
+            Assert.True(result.Success, result.Output);
+            Assert.Contains(@$"<PackageVersion Include=""X"" Version=""2.0.0"" />", File.ReadAllText(Path.Combine(pathContext.SolutionRoot, "Directory.Packages.props")));
+
+            // Checking that version metadata is not added to the project files.
+            Assert.Contains(@$"Include=""X""", File.ReadAllText(Path.Combine(projectADirectory, "projectA.csproj")));
+            Assert.DoesNotContain(@$"Include=""X"" Version=""1.0.0""", File.ReadAllText(Path.Combine(projectADirectory, "projectA.csproj")));
+            Assert.DoesNotContain(@$"Include=""X"" Version=""2.0.0""", File.ReadAllText(Path.Combine(projectADirectory, "projectA.csproj")));
+        }
+
+        [Fact]
+        public async Task AddPackageVersionToCorrectItemGroupInPropsFile()
+        {
+            using var pathContext = new SimpleTestPathContext();
+
+            // Set up solution
+            var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+            var projectNameA = "projectA";
+            var projectA = XPlatTestUtils.CreateProject(projectNameA, pathContext, "net5.0");
+
+            const string version = "1.0.0";
+            const string packageX = "X";
+
+            var packageFrameworks = "net5.0";
+            var packageX100 = XPlatTestUtils.CreatePackage(packageX, version, frameworkString: packageFrameworks);
+
+            await SimpleTestPackageUtility.CreateFolderFeedV3Async(
+                    pathContext.PackageSource,
+                    PackageSaveMode.Defaultv3,
+                    packageX100);
+
+            var propsFile = @$"<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include=""SomeFile"" />
```

Comment:
The content was actually only added to the Props file in this test because of @jeffkl 's comment here: https://github.com/NuGet/NuGet.Client/pull/4700#discussion_r908818570 . Hopefully that helps clarify things but please let me know if not.

Reply:
I see. 

I guess it could only be done for some specific tests and not all, but I'll let you make a call on that.